versions/kruise/1.3.0/templates/manager.yaml

```diff
@@ -197,6 +197,23 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: spec.nodeName
+        {{- range $key, $value := .Values.daemon.env }}
```

Comment:
{{- if .Values.daemon.env }}
{{- toYaml .Values.daemon.env | nindent 8 }}
{{- end }}

Reply:
I tried your suggestions, but I have the following error:

`coalesce.go:220: warning: cannot overwrite table with non table for kruise.daemon.env (map[])` 

The problem seems to be due to the name `env`.  Replace `env` with `extraEnvs` it works.
I pushed the changes.